README.md

```diff
@@ -25,7 +25,7 @@
 # 📖 Blog App <a name="about-project"></a>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215249168-2dbb74d7-20ff-48a1-b6c5-eb64638c9817.png)

**[Optional]**

- [ ] These are not your key features :smile: Key features mean what your application does such as **create posts, like posts, or comment on posts** :+1:

Reply:
Thanks